Library/Formula/haproxy.rb

```diff
@@ -16,13 +14,14 @@ class Haproxy < Formula
   depends_on "pcre"
 
   def install
-    args = ["TARGET=generic",
-            "USE_KQUEUE=1",
-            "USE_POLL=1",
-            "USE_PCRE=1",
-            "USE_OPENSSL=1",
-            "USE_ZLIB=1",
-            "ADDLIB=-lcrypto",
+    args = %w[
```

Comment:
Ah. Is it only ` args = %W[ ` when you do something like ` --prefix=#{prefix} ` or similar?